manifests/java.yml

```diff
@@ -744,7 +744,11 @@ tests/:
         spring-boot-payara: bug (user block results in 500 response)
         vertx3: v1.7.0
         vertx4: v1.7.0
-      Test_Suspicious_Request_Blocking: missing_feature
+      Test_Suspicious_Request_Blocking:
+        '*': v1.6.0
+        akka-http: bug
```

Comment:
It's a well documented bug? or do we have any clue? IMHO it's a good idea to add more info about this if we have it

Reply:
I did not look into it in depth so far, I'm not sure if it's lack of support or bug.